common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -56,7 +56,8 @@ case class ActivationMessage(
     activationNamespace: EntityPath,
     rootControllerIndex: InstanceId,
     content: Option[JsObject],
-    cause: Option[ActivationId] = None)
+    cause: Option[ActivationId] = None,
+    tracingMetadata: Option[Map[String, String]] = None)
```

Comment:
how big might this map get?

in general this is going to be a problem - we have a standing issue that the control and data plane are conflated.
because these message pass over kafka, and carry the user payload, every bit added to the control is one less bit for the user payload; in other words, we have a bug today that a user cannot send a 1MB payload through and taking away more bits makes it worse... hence the question.

for data, I expect that message to flow externally through a redis for example.